api/src/main/java/com/netflix/iceberg/expressions/Expression.java

```diff
@@ -94,6 +94,44 @@ public Operation flipLR() {
           throw new IllegalArgumentException("No left-right flip for operation: " + this);
       }
     }
+
+    @Override
+    public String toString() {
+      switch (this) {
+        case TRUE:
+          return "true?";
```

Comment:
True and false aren't tests, like "x == true", they are expressions that always evaluate to true or false. These are needed because users will occasionally pass them when generating queries and for expressions that can be simplified. When `i < 91828349178239487` is bound to an integer field, the value is above Integer.MAX_VALUE so we can replace the predicate with `true`.